app/controllers/evms_controller.rb

```diff
@@ -94,7 +99,7 @@ def default_calcetc
   end
 
   def default_setting(setting_name, defaultvalue)
-    value = Setting.plugin_redmine_issue_evm[setting_name].blank? ? defaultvalue : Setting.plugin_redmine_issue_evm[setting_name].to_f
+    Setting.plugin_redmine_issue_evm[setting_name].blank? ? defaultvalue : Setting.plugin_redmine_issue_evm[setting_name].to_f
```

Comment:
__[rubocop]__
* Line is too long. [126/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)